extensions/libarchive/MergeContent.cpp

```diff
@@ -271,6 +281,16 @@ bool MergeContent::processBin(core::ProcessContext *context, core::ProcessSessio
       return false;
     }
     session->putAttribute(mergeFlow, BinFiles::FRAGMENT_COUNT_ATTRIBUTE, std::to_string(bin->getSize()));
+
+    std::unique_ptr<AttributeMerger> attributeMerger;
+    if (attributeStrategy_ == ATTRIBUTE_STRATEGY_KEEP_COMMON)
+      attributeMerger = std::unique_ptr < AttributeMerger > (new KeepOnlyCommonAttributesMerger(session, bin->getFlowFile(), mergeFlow));
+    else if (attributeStrategy_ == ATTRIBUTE_STRATEGY_KEEP_ALL_UNIQUE)
+      attributeMerger = std::unique_ptr < AttributeMerger > (new KeepAllUniqueAttributesMerger(session, bin->getFlowFile(), mergeFlow));
```

Comment:
Minor, but padding `<`-s from both sides makes it seem like they are relational operators.
See Templates and Casts in the google styleguide we follow:
https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace

Reply:
Good point, I'll also remove them at line 268-272 in this case.